### DISCUSSION TITLE
中国移动云盘/139云盘/和彩云盘（个人云） Failed init storage:鉴权失败

### DISCUSSION THREAD

**Discussion by dujiayao** [AlistGo/alist#4589] [General]
中国移动云盘/139云盘/和彩云盘 （个人云） Failed init storage:鉴权失败
2023年6月8日开始，新抓取的Cookie都是 鉴权失败。看了下移动云盘更新了，cookie结尾部分增加了时间戳。
同时，发现移动云盘新增了类似阿里云盘的开放平台：http://open.yun.139.com/
<img width="960" alt="Failed" src="https://github.com/alist-org/alist/assets/35137996/08049fbc-ce80-461c-9a51-3847bfd0760c">
<img width="915" alt="Cookie" src="https://github.com/alist-org/alist/assets/35137996/0fab3029-0be8-42e8-92e7-2381c33c09fa">
<img width="407" alt="Root ID" src="https://github.com/alist-org/alist/assets/35137996/b81220c3-d0d4-4078-beae-771a1a8f647e">

**Comment by Edison-TT** [comment]
确实鉴权失败，有解决方案部

**Comment by paullamlam** [comment]
一样，等待技术救援中。

**Comment by anwen-anyi** [comment]
等下个版本修复 
https://github.com/alist-org/alist/commit/e23f00f349706ad646390a812da20733567f9031
https://github.com/alist-org/alist/commit/a66b0e015168f73d1b24476fe7bb683538e40ae9
https://github.com/alist-org/alist/commit/6cbd2532ccf42f48051927dbaec4e7d2d100c578
